### ISSUE TITLE
Accessibility : Fast pass issue. Dropdown with multi select and required properties not allowing aria attributes i.e  aria-required="true"

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#14355]
### Environment Information

- **Package version(s)**: "office-ui-fabric-react": "^7.123.5",
- **Browser and OS versions**: Chrome and Windows 10

### Describe the issue:
Using Multi select option for dropdown. Fast pass is failing stating aria attributes are not allowed i.e aria-required="true".

#### Actual behavior:
Using Multi select option for dropdown. Fast pass is failing stating aria attributes are not allowed i.e aria-required="true".

#### Expected behavior:
Using Multi select option for dropdown. Fast paas should pass.

### If applicable, please provide a codepen repro:
https://codesandbox.io/s/angry-raman-dd5p2?file=/src/index.js
<img width="1330" alt="multiselectScreenshot_1" src="https://user-images.githubusercontent.com/48669559/89407064-c36d8b80-d73b-11ea-86aa-676dcce3462f.png">
<img width="746" alt="multiselectScreenshot_2" src="https://user-images.githubusercontent.com/48669559/89407088-c8323f80-d73b-11ea-83c4-3ef34db6ed2c.png">

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by khmakoto** [comment]
Hi @Sandhya, I can confirm that this is reproable in both your codesandbox and our website. I'll work on a fix as soon as I can.
